pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/Style.java

```diff
@@ -0,0 +1,26 @@
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+public class Style {
+
+
+    public static TextFlow highlight(String text, String filter ) {
+        int filterIndex = text.toLowerCase().indexOf(filter.toLowerCase());
+        Text textBefore = new Text(text.substring(0, filterIndex));
+        Text textAfter = new Text(text.substring(filterIndex + filter.length()));
+        Text textFilter = new Text(text.substring(filterIndex,  filterIndex + filter.length())); //instead of "filter" to keep all "case sensitive"
+        textFilter.setFill(Color.ORANGE);
+        textFilter.setFont(Font.font("Helvetica", FontWeight.BOLD, 12));
```

Comment:
This styling is ad hoc and isn't consistent with the rest of the designer. Don't set the font